src/Sql/dbo/User Defined Types/WebauthnCredentialRotationDataType.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TYPE [dbo].[WebauthnCredentialRotationDataType] AS TABLE(
```

Comment:
@rkac-bw can help work with you on this, but we'd like to stop using custom SQL types.

Reply:
So, something more like is done for ciphers?:
https://github.com/bitwarden/server/blob/a1d609b208f37f45bd8cf8d7c22769cd0455b281/src/Infrastructure.Dapper/Vault/Repositories/CipherRepository.cs#L327-L381

Reply:
Doesn't even need to be that advanced with bulk copy. There's a JSON input that can be used. Robert will have a recent example.

Reply:
Adjusted the implementation based on Robert's example. Thanks!